app/templates/index.html

```diff
@@ -94,11 +94,10 @@
     {% if current_user.role != 'supplier' %}
       <div class="padding-bottom-small">
         <p>
-          <a href="/suppliers/create" class="top-level-link">
-            Create a supplier account
+          <a href="/suppliers/supply" class="top-level-link">
```

Comment:
I don't think I understand why the URL needs to change. `/suppliers/supply` feels super redundant and I don't think `supply` as a verb really encapsulates what they're doing. The URL doesn't let them supply anything (by itself). They're creating an account, and that's the content they get on the page. Is `create-account` better?

Reply:
The `/suppliers/create` page still exists.  The new page will not always let them create an account - it's the "Become a supplier" page, which might only let them sign up for email notifications if there are no frameworks "coming" or "open" (eg like right now).

I'm not completely sold on the URL - if you can think of something better for this page (see screenshots at https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/741) then let us know.

But Laurence, Constance and Chelsea decided on `/supply` for the ticket, and we have no product person at the moment to OK any proposed changes.

Reply:
Ah I see! I'll have a think.

Reply:
Yeah I wasn't sure about `supply` but I couldn't think of anything better.